code/5-demographics/inference/run_inference_BR.sbatch

```diff
@@ -0,0 +1,15 @@
+#SBATCH --job-name=Demographic_Inference
+#SBATCH --nodes=1
+#SBATCH --cpus-per-task=4
+#SBATCH --mem=15GB
+#SBATCH --time=24:00:00
+
+module purge
+module load python3/intel/3.6.3
+source /beegfs/vp1274/twitter/venvs/twitter-env/bin/activate
+cd /home/vp274/
+
+python demographic_inference.py --images_dir "/scratch/spf248/twitter/data/classification/BR/profile_pictures_sam/" \
+--resized_images_dir "/scratch/spf248/twitter/data/classification/BR/profile_pictures_resized_random/" \
+--output_dir "/scratch/spf248/twitter/data/classification/BR/inference_results" \
+--country "BR"
```

Comment:
you could just have one batch file with `COUNTRY_CODE=$1` after the SBATCH arguments and assign `${COUNTRY_CODE}` to the `--country` argument, right?

Reply:
I wasn't sure how would we launch job array for this. My idea was to run separate jobs for each of them. Especially in future if the code needs to be re-run for a single country, this would be much more extendable, so I followed SOLID principles here.

Reply:
Btw I can make the script to accept the parameter.

Reply:
Ah right, with job arrays, it might be trickier. sounds good!